Sprint-3/quote-generator/index.html

```diff
@@ -1,15 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
-    <script defer src="quotes.js"></script>
-  </head>
-  <body>
-    <h1>hello there</h1>
-    <p id="quote"></p>
-    <p id="author"></p>
-    <button type="button" id="new-quote">New quote</button>
-  </body>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Quote Generator App</title>
+  <link rel="stylesheet" href="style.css">
+  <script defer src="quotes.js"></script>
+</head>
+<body>
+  <div class="quote-container">
+    <h1>Random Quote Generator</h1>
+    <p id="quote" class="quote-text"></p>
+    <p id="author" class="quote-author"></p>
+    <button type="button" id="new-quote" class="btn">New Quote</button>
+    <label>
+      <input type="checkbox" id="auto-play"> Auto-Play
+    </label>
+    <p id="auto-play-status"></p>
```

Comment:
It would be better to show "Auto-Play: OFF" initially (to be consistent with the unchecked "Auto-Play" checkbox).